### ISSUE TITLE
YouPHPTube - rtsp stream doesnt work

### ISSUE THREAD

**Issue by tobit6795** [turnkeylinux/tracker#1396]
i assume that the api isnt working correct

i m using https + base auth OOTB

on the other side, i have a plain installation with http and w/o baseauth, works...

**Comment by JedMeister** [comment]
Thanks for your bug report.

I did make some significant improvements to our YouPHPTube appliance late last year (see https://github.com/turnkeylinux-apps/youphptube/pull/4). However, unfortunately, I didn't get a chance to test them exhaustively and at that point was hoping to have our upcoming v16.0 release ready "any day soon".

Alas it wasn't to be, and v16.0 is still not released... We're close, but we're currently making some further improvements to our backup infrastructure and processes. That's holding things up at the moment, but will be a serious improvement once completed.

None of that is probably any real consolation to you right now, but perhaps looking at the code I've included might assist you to get things working as they should?
